Formula/gnuradio.rb

```diff
@@ -1,9 +1,9 @@
 class Gnuradio < Formula
   desc "SDK providing the signal processing runtime and processing blocks"
-  homepage "https://gnuradio.squarespace.com/"
-  url "https://gnuradio.org/releases/gnuradio/gnuradio-3.7.9.1.tar.gz"
-  sha256 "9c06f0f1ec14113203e0486fd526dd46ecef216dfe42f12d78d9b781b1ef967e"
-  revision 1
+  homepage "http://gnuradio.org/"
+  url "http://gnuradio.org/releases/gnuradio/gnuradio-3.7.10.1.tar.gz"
```

Comment:
Changed to TLS anyway, it works and maybe some day their server will serve file directly on SSL.